.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -35,3 +36,4 @@ Full release instructions are at: [RELEASE.md](/sigstore/protobuf-specs/blob/mai
 - [ ] [Ruby](https://rubygems.org/gems/sigstore_protobuf_specs)
 - [ ] [Rust](https://crates.io/crates/sigstore_protobuf_specs)
 - [ ] [Typescript](https://www.npmjs.com/package/@sigstore/protobuf-specs)
+- [ ] [Service Buidler Container](https://github.com/sigstore/protobuf-specs/pkgs/container/protobuf-specs-service-builder)
```

Comment:
```suggestion
- [ ] [Service Builder Container](https://github.com/sigstore/protobuf-specs/pkgs/container/protobuf-specs-service-builder)
```

Reply:
Can you also add some documentation somewhere about what this is for?

Reply:
hrmm, alright I'll find somewhere to put it